web_chatter_position/static/src/scss/chatter_form_adjustments.scss

```diff
@@ -0,0 +1,9 @@
+.o_form_view_container {
+    .o_form_sheet_bg .o_form_sheet {
+        max-width: none !important;
+        width: auto !important;
+    }
+    .o-mail-Form-chatter {
+        max-width: none;
```

Comment:
While this class name is actually in use, it does not make a difference when I test locally (on runbot, the chatter is full width, but that's probably because of web_responsive).
![image](https://github.com/user-attachments/assets/5dbce055-984d-43c3-9d3c-58a60b9fddd8)

Reply:
Any opinion on the suggestions I put in my PR for your branch?